app/src/pages/playground/ModelConfigButton.tsx

```diff
@@ -322,29 +347,30 @@ function ModelConfigDialogContent(props: ModelConfigDialogContentProps) {
   );
 
   return (
-    <View padding="size-200" overflow="auto">
-      <Form>
-        <Flex direction="column" gap="size-200">
-          <ModelProviderPicker
-            provider={instance.model.provider}
-            query={query}
-            onChange={updateProvider}
-          />
-          {instance.model.provider === "AZURE_OPENAI" ? (
-            <AzureOpenAiModelConfigFormField instance={instance} />
-          ) : (
-            <ModelPicker
-              modelName={instance.model.modelName}
-              provider={instance.model.provider}
-              query={query}
-              onChange={onModelNameChange}
-            />
-          )}
-          <Suspense>
-            <InvocationParametersForm instanceId={playgroundInstanceId} />
-          </Suspense>
-        </Flex>
-      </Form>
-    </View>
+    <form
+      onSubmit={(e) => {
```

Comment:
Can this close the dialog? Would be nice to close on enter

Reply:
good call

Reply:
looks like the form fields arent' triggering submission anyways going to just remove and punt